tests/Fixer/FunctionNotation/VoidReturnFixerTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\FunctionNotation;
+
+use PhpCsFixer\Test\AbstractFixerTestCase;
+
+/**
+ * @author   Mark Nielsen
+ *
+ * @internal
+ *
+ * @requires PHP 7.1
+ * @covers   \PhpCsFixer\Fixer\FunctionNotation\VoidReturnFixer
+ */
+final class VoidReturnFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @dataProvider provideFixCases
+     *
+     * @param string      $expected
+     * @param null|string $input
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixCases()
```

Comment:
I think fixers should assume the processed code is correct. If a method like in this example is fixed, it will help the developer notice the error as he will see an unexpected `void` in the diff (or his test suite will fail). The fixer is risky so users are aware they should double check their codebase after a PHP CS Fixer run.